manager/src/test/scala/org/broadinstitute/transporter/db/DbClientSpec.scala

```diff
@@ -23,7 +23,11 @@ import org.broadinstitute.transporter.transfer.api.{
   TransferDetails,
```

Comment:
some of the testing seems brittle, if future iterations of this test are to be improved it mkght makes sense to redesign some of the logic such as hard coding the number of queues